lib/styles/default/text_inputs.dart

```diff
@@ -35,10 +37,25 @@ class DefaultTextInputStyles {
           containerStyle: CommonContainerModel(
             paddingLeft: 0.05,
             paddingRight: 0.048,
+            height: 0.5,
           ),
           path: icon,
           color: AppColors.blackLight,
         ),
+        onTap: () => Get.put(SignUpController()).passwordEmpty(),
```

Comment:
You can pass the attribute to the style instead of reading it from the controller like this